.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - DOCKER_VERSION=1.12.6 ENABLE_SWARM=1 UPLOAD_SONATYPE=1
     - DOCKER_VERSION=1.13.1 ENABLE_SWARM=1
     - DOCKER_VERSION=17.03.0~ce ENABLE_SWARM=1
+    - DOCKER_VERSION=17.04.0~ce ENABLE_SWARM=1
```

Comment:
I'm not really sure to be honest. I am not crazy about throwing out tests for old versions as we should try to ensure we are still compatible there, but I don't want to miss things in new versions either. On the other hand the more versions we add, the longer Travis builds take.

Perhaps we could start by continually adding new versions and stretching out the number of versions we test to be more than six, and then we can reduce the number when/if we find the long build times are too cumbersome.